test/controllers/sessions_controller_test.rb

```diff
@@ -33,10 +33,14 @@ class SessionsControllerTest < ActionController::TestCase
   end
 
   test 'the ldap server has cant find the user data' do
-    User.stub :create_or_update_from_active_directory, ->(arg) { raise ChapmanIdentityNotFound } do
-      Net::LDAP.stub_any_instance(:bind, true) do
-        post :create, user: { email: 'kerr105@chapman.edu', password: 'blahblahbalah' }
-        assert response.body.include?('Invalid email or password.')
+    # test for invalid identity info, VCR needs a valid user due to how authenticate! method is written
+    # even though testing for an invalid user 
+    VCR.use_cassette(:lookup_kerr105) do
+      User.stub :create_or_update_from_active_directory, ->(arg) { raise ChapmanIdentityNotFound } do
+        Net::LDAP.stub_any_instance(:bind, true) do
+          post :create, user: { email: 'kerr105@chapman.edu', password: 'blahblahbalah' }
+          assert response.body.include?('Invalid email or password.')
```

Comment:
This still needs to be updated to check for "Invalid identity info." instead of "Invalid email or password".